spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/rdd/CassandraRDDSpec.scala

```diff
@@ -627,4 +632,28 @@ class CassandraRDDSpec extends FlatSpec with Matchers with SharedEmbeddedCassand
     result should have length 0
   }
 
+  it should "suggest mixed case keyspaces if they exist" in {
+    try {
```

Comment:
Matchers contain a nice construct for this:
val exception = the [Exception] thrown by { some code } 
exception.message should ...